packages/iris-grid/src/IrisGridTestUtils.ts

```diff
@@ -92,7 +92,7 @@ class IrisGridTestUtils {
   makeTable({
     columns = this.makeColumns(),
     size = 1000000000,
-    sort = [],
+    sort = [] as Sort[],
     layoutHints = {} as LayoutHints,
   } = {}): Table {
```

Comment:
This is fine as is, and consistent with what's there.

These days I would prefer to detail the type rather than requiring casting, e.g.
```
  makeTable({
    columns = this.makeColumns(),
    size = 1000000000,
    sort = [],
    layoutHints = {},
  }: {
    columns?: Column[];
    size?: number;
    sort?: readonly Sort[];
    layoutHints?: LayoutHints;
  } = {}): Table {
```

Or even if you define the type prior, and export it so it can be used in another file, e.g.
```
export type MakeTableOptions = {
  columns: Column[];
  size: number;
  sort: readonly Sort[];
  layoutHints: LayoutHints;
}

  makeTable({
    columns = this.makeColumns(),
    size = 1000000000,
    sort = [],
    layoutHints = {},
  }: Partial<MakeTableOptions> = {}): Table {
```